docker-compose.link.yaml

```diff
@@ -9,10 +9,7 @@ services:
       context: .
       target: development
     command: >
-      sh -c "
-        scripts/wait-for/wait-for ffc-demo-payment-postgres:5432 --
-        npm run migrate &&
-        npm run start:watch"
+      sh -c "npm run start:watch"
```

Comment:
This is already the command in the Dockerfile so no need for any command override.